src/framework/ui/checkbox/checkbox.component.tsx

```diff
@@ -54,34 +59,46 @@ export class CheckBox extends React.Component<Props> {
     }
   };
 
+  private getSelectSize = (indeterminate: boolean, width: number, height: number): StyleType => {
+    if (indeterminate) {
+      return {
+        width: width - width * 0.18,
+        height: height * 0.18,
+      };
+    } else {
+      return {
+        width: width * 0.5,
+        height: height * 0.5,
+      };
+    }
+  };
+
   private getComponentStyle = (style: StyleType): StyleType => {
     const {
       textColor,
       textFontSize,
       textFontWeight,
-      textMarginLeft,
-      selectWidth,
-      selectHeight,
-      selectBackgroundColor,
+      textLineHeight,
+      checkMarkColor,
```

Comment:
background, foreground or tint?

Reply:
Also while there is no special parameter for `indeterminate` icon, I think it should have `icon` prefix

Reply:
Just variable name from Eva Mapping doc. Changing the names of such variables should be discussed with the parallel team.